infrastructure/stacks/service_etl/functions/service_etl/service_etl.py

```diff
@@ -9,6 +9,10 @@
 from requests_aws4auth import AWS4Auth
 from elasticsearch import Elasticsearch, RequestsHttpConnection, helpers
 import time
+import logging
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
Is it possible to extract the logging level out into an environment variable and have it defined in the profile files so we can easily tweak it?